packages/Calendar/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@tds/community-calendar",
```

Comment:
Please double check the component name with your designer, I believe it should be `@tds/community-calendar-picker`